lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -30,22 +32,23 @@
             className: 'team-members',
             template: _.template(teamMembershipDetailsTemplate),
```

Comment:
Is the thought that we don't care whether someone uses `_.template()` or `HtmlUtils.template()`.  They just have to use the latter if they want access to HtmlUtils in the template?